tests/makefile

```diff
@@ -0,0 +1,19 @@
+new: venv refresh
```

Comment:
It's better to use install instead of new... more common wording for people to know how to install it..